packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/edit-connector/edit-connector.controller.js

```diff
@@ -37,6 +38,12 @@ export default class EditConnectorCtrl {
     this.messages = this.messageHandler.getMessages();
   }
 
+  areRequiredFieldsFilled() {
+    return this.requiredFields.every(
+      ({ name }) => this.model[name] != null && this.model[name] !== '',
```

Comment:
**suggestion**: Isn't this enough?
```suggestion
      ({ name }) => this.model[name],
```

Reply:
No, if a required field is set to 0, this will  incorrectly return `false` :
```
let model = []
let requiredFields = [{name: 'number'}]

model[number] = 6;
requiredFields.every(({name}) => model[name]) // return true

model[number] = 0;
requiredFields.every(({name}) => model[name]) // return false, even if the field is correct
```